packages/brokers/package.json

```diff
@@ -0,0 +1,88 @@
+{
+	"name": "@discordjs/brokers",
+	"version": "0.1.0-dev",
+	"description": "Powerful set of message brokers",
+	"scripts": {
+		"test": "vitest run",
+		"build": "unbuild",
+		"lint": "prettier --check . && TIMING=1 eslint src __tests__ --ext mjs,js,ts",
+		"format": "prettier --write . && TIMING=1 eslint src __tests__ --ext mjs,js,ts --fix",
+		"fmt": "yarn format",
+		"docs": "downlevel-dts dist docs/dist --to=3.7 && docgen -i src/index.ts -c docs/index.json -o docs/docs.json --typescript && api-extractor run --local",
+		"prepack": "yarn lint && yarn test && yarn build",
+		"changelog": "git cliff --prepend ./CHANGELOG.md -u -c ./cliff.toml -r ../../ --include-path 'packages/proxy/*'",
+		"release": "cliff-jumper"
+	},
+	"main": "./dist/index.cjs",
+	"module": "./dist/index.mjs",
+	"typings": "./dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.cjs",
+		"types": "./dist/index.d.ts"
+	},
+	"directories": {
+		"lib": "src",
+		"test": "__tests__"
+	},
+	"files": [
+		"dist"
+	],
+	"contributors": [
+		"Crawl <icrawltogo@gmail.com>",
+		"Amish Shah <amishshah.2k@gmail.com>",
+		"SpaceEEC <spaceeec@yahoo.com>",
+		"Vlad Frangu <kingdgrizzle@gmail.com>",
+		"Antonio Roman <kyradiscord@gmail.com>",
+		"DD <didinele.dev@gmail.com>"
+	],
+	"license": "Apache-2.0",
+	"keywords": [
+		"discord",
+		"api",
+		"rest",
+		"proxy",
+		"discordapp",
+		"discordjs"
+	],
```

Comment:
Doesn't look like these were changed. Should they?

Reply:
yup. i forgot